stable/magic-namespace/templates/tiller-deployment.yaml

```diff
@@ -30,7 +30,7 @@ spec:
     spec:
       serviceAccountName: tiller
       containers:
-        - name: {{ .Chart.Name }}
+        - name: tiller
```

Comment:
I'm not opposed to this, but it might be better as a separate PR since it's otherwise unrelated to the TLS capability.

Reply:
Thanks, I've removed this and the service account default changes to #12105 for consideration after this PR is merged.